eng/templates/stages/release.yml

```
@@ -86,17 +86,13 @@ stages:
     - name: sdkArtifactFileName
       value: dotnet-sdk-*.tar.gz
 
-  # GitHub release is skipped for 6.0/7.0
-  # For dry run, we only create a draft release
+  # GitHub release is skipped for 6.0/7.0 and dry runs
   - name: createGitHubRelease
     ${{ if eq(parameters.createGitHubRelease, 'auto') }}:
-      ${{ if or(eq(parameters.dotnetMajorVersion, '6.0'), eq(parameters.dotnetMajorVersion, '7.0')) }}:
+      ${{ if or(eq(parameters.isDryRun, parameters.dotnetMajorVersion, '6.0'), eq(parameters.dotnetMajorVersion, '7.0')) }}:
```

I don't think this syntax is correct unless I'm missing something. It looks like three arguments are being passed to the `eq` function:

```
eq(parameters.isDryRun, parameters.dotnetMajorVersion, '6.0')
```

---

Oops, should have been this:

```suggestion
      ${{ if or(parameters.isDryRun, eq(parameters.dotnetMajorVersion, '6.0'), eq(parameters.dotnetMajorVersion, '7.0')) }}:
```